sys/hacl/build.rs

```diff
@@ -24,35 +30,44 @@ fn includes(home_dir: &Path) -> Vec<String> {
     ]
 }
 
-fn append_simd128_flags(flags: &mut Vec<String>) {
-    // Platform detection
-    if cfg!(simd128) {
-        flags.push("-DSIMD128".to_string());
-        if cfg!(any(target_arch = "x86", target_arch = "x86_64")) {
-            flags.push("-msse4.1".to_string());
-            flags.push("-msse4.2".to_string());
-            flags.push("-mavx".to_string());
-        }
+fn append_simd128_flags(flags: &mut Vec<String>, is_bindgen: bool) {
+    if is_bindgen || cfg!(all(any(target_arch = "x86", target_arch = "x86_64"), not(target_env = "msvc"))) {
+        flags.push("-msse4.1".to_string());
+        flags.push("-msse4.2".to_string());
+        flags.push("-mavx".to_string());
     }
 }
 
-fn append_simd256_flags(flags: &mut Vec<String>) {
-    // Platform detection
-    if cfg!(simd256) {
-        flags.push("-DSIMD256".to_string());
+fn append_simd256_flags(flags: &mut Vec<String>, is_bindgen: bool) {
+    if is_bindgen || cfg!(not(target_env = "msvc")) {
         flags.push("-mavx2".to_string());
     }
 }
 
-#[cfg(not(windows))]
-fn create_bindings(home_dir: &Path) {
-    let mut clang_args = includes(home_dir);
-    append_simd128_flags(&mut clang_args);
-    append_simd256_flags(&mut clang_args);
+fn create_bindings(platform: Platform, home_dir: &Path) {
```

Comment:
Let's add a note to the readme that in order to build this the libclang path needs to be set correclty.

Reply:
Maybe I misunderstood the comment, but I see Cargo compile `clang-sys` to use it for `bindgen`

Reply:
I filed #61 as a follow up for this.